config/environments/staging.rb

```diff
@@ -47,7 +47,8 @@
 
   # Force all access to the app over SSL, use Strict-Transport-Security,
   # and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
why is this needed for staging?

Reply:
I attempted to explain in the PR description